### PR TITLE
Add objc.io books

### PR DESCRIPTION
**Pull request by kuquert**
Add the three books by objc.io. Including:
- [Advanced Swift](https://www.objc.io/books/advanced-swift/)
- [Functional Swift](https://www.objc.io/books/functional-swift/)
- [Core Data](https://www.objc.io/books/core-data/)
## Project URL
https://www.objc.io/books/

## Description
Three amazing Swift books that should be on the list.
 
## Checklist

- [x] Only one project/change is in this pull request
- [x] Addition in chronological order (bottom of category)
- [x] Supports iOS 9 or later
- [x] Supports Swift 3
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

Some of the checks don`t apply to books.

### CODE REVIEW

**Comment by danger-awesome-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Found 1 link issue</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Too many changes (when adding, please keep it to one project per Pull Request)</td>
    </tr>
  </tbody>
</table>

#### Link issue by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
2077 | [302](https://httpstatuses.com/302) | https://swiftlang.ng.bluemix.net/#/repl redirects to<br>https://swiftlang.ng.bluemix.net/verify


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -2352,6 +2352,10 @@ Most of these are paid services, some have free tiers.
 * [Hello Swift! by Tanmay Bakshi with Lynn Beighley](https://www.manning.com/books/hello-swift) :large_orange_diamond:
 * [OS Development with Swift by Craig Grummitt](https://www.manning.com/books/ios-development-with-swift) :large_orange_diamond:
 * [Anyone Can Create an App by Wendy L. Wise](https://www.manning.com/books/anyone-can-create-an-app) :large_orange_diamond:
+* [Advanced Swift by Chris Eidhof, Ole Begemann, and Airspeed Velocity](https://www.objc.io/books/advanced-swift/) :large_orange_diamond:
+* [Functional Swift by Chris Eidhof, Florian Kugler, and Wouter Swierstra](https://www.objc.io/books/functional-swift/) :large_orange_diamond:
+* [Core Data by Florian Kugler and Daniel Eggert](https://www.objc.io/books/core-data/) :large_orange_diamond:
+
```
Comment (lfarah) [inline README.md]: Can you please delete this extra space?

**Comment by kuquert** [comment]
Updated

**Comment by dkhamsing** [comment]
Thank you for contributing to `awesome-ios` 😄

**Pull request merged**